test/playwright.config.ts

```diff
@@ -8,6 +8,7 @@ import path from "path"
 //   yarn test:e2e --workers 1        # Run with one worker
 //   yarn test:e2e --project Chromium # Only run on Chromium
 //   yarn test:e2e --grep login       # Run tests matching "login"
+//   PWDEBUG=1 yarn test:e2e          # Run Playwright inspector
```

Comment:
saves time from having to look this up each time 😂

Reply:
100% lol same reason I added the other comments I am always forgetting the flags